lib/src/dialog.js

```diff
@@ -22,6 +22,7 @@
  */
 
 var dialog = require('dialog');
+var Promise = require('bluebird');
```

Comment:
'Promise' is defined in a future version of JavaScript. Use a different variable name to avoid migration issues.